admin/securedrop_admin/__init__.py

```diff
@@ -85,7 +85,7 @@ def lookup_fqdn(self, fqdn, dns=None):
                 cmd = 'torify ' + cmd
             cmd += ' ' + (dns and dns or '8.8.8.8')
             try:
-                result = subprocess.check_output(cmd, shell=True,
+                result = subprocess.check_output(cmd,
```

Comment:
Since `cmd` is a string, don't we need to convert to a list using e.g. `cmd.split(' ')` or similar?

Reply:
Yes, we do. Good catch!